bwc-test/build.gradle

```diff
@@ -131,14 +145,17 @@ def String extractVersion(versionStr) {
                 node.setting("plugins.security.ssl.http.pemcert_filepath", "esnode.pem")
                 node.setting("plugins.security.ssl.http.pemkey_filepath", "esnode-key.pem")
                 node.setting("plugins.security.ssl.http.pemtrustedcas_filepath", "root-ca.pem")
+                // node.setting("plugins.security.ssl.http.truststore_filepath", "truststore.jks")
+                // node.setting("plugins.security.ssl.http.truststore_password", "password")
                 node.setting("plugins.security.allow_unsafe_democertificates", "true")
                 node.setting("plugins.security.allow_default_init_securityindex", "true")
-                node.setting("plugins.security.authcz.admin_dn", "CN=kirk,OU=client,O=client,L=test,C=de")
-                node.setting("plugins.security.audit.type", "internal_elasticsearch")
+                node.setting("plugins.security.authcz.admin_dn", "\"CN=kirk,OU=client,O=client,L=test,C=de\"")
```

Comment:
I don't think this change is actually necessary. I plan to slim down this PR to only the changes that are required. I opened up a first Draft to show the bwc tests working with the security plugin installed which included other minor changes I made when trying to get the bwc tests to run. I'm aiming to get this out of draft later today.

FYI there are 2 different ways that lists are defined in opensearch.yml and I can't tell if they are equivalent.

1. With an array

```
plugins.security.restapi.roles_enabled: ["all_access", "security_rest_api_access"]
```

2. With a hyphenated list:

```
plugins.security.authcz.admin_dn:
  - CN=kirk,OU=client,O=client,L=test, C=de
```

When entering the node settings here, I couldn't tell if this value was being entered correctly because its expected to be a hyphenated list.